style.css

```diff
@@ -0,0 +1,780 @@
+.about-headline {
+  margin-bottom: 40px;
+  padding-top: 100px;
+}
+
+.headline-welcome {
+  color: #ec5242;
+}
+
+.headline-welcome h2 {
+  font-size: 20px;
+  font-weight: 400;
+}
+
+.headline-welcome h1 {
+  font-size: 40px;
+  font-weight: 900;
+  width: 50%;
+}
+
+.headline-welcome .title {
+  background-image: url('https://images.unsplash.com/photo-1620120966883-d977b57a96ec?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTh8fGRpZ2l0YWwlMjBtYXJrZXRpbmclMjBvcmFuZ2V8ZW58MHx8MHx8&auto=format&fit=crop&w=600&q=60');
+  background-size: auto;
+  background-position: center;
+  background-repeat: repeat;
+  -webkit-background-clip: text;
+  background-clip: text;
+  color: transparent;
+  width: 100%;
+}
+
+.about-headline-welcome {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  justify-content: normal;
+  align-items: center;
+  align-content: normal;
+  color: #ec5242;
+}
+
+.about-headline-welcome h2 {
+  display: block;
+  flex-grow: 0;
+  flex-shrink: 1;
+  flex-basis: auto;
+  align-self: auto;
+  order: 0;
+  font-size: 20px;
+  font-weight: 400;
+}
+
+.about-headline-welcome h1 {
+  display: block;
+  text-align: center;
+  flex-grow: 0;
+  flex-shrink: 1;
+  flex-basis: auto;
+  align-self: auto;
+  order: 0;
+  font-size: 40px;
+  font-weight: 900;
+}
+
+.headline-text {
+  border: white solid 2px;
+  margin: 30px 0;
+  padding: 16px;
+  font-size: 12px;
+}
+
+.about-headline-text {
+  border: lightgray solid 1px;
+  background: white;
+  text-align: center;
+  align-self: center;
+  margin: 30px 0;
+  padding: 16px;
+  font-size: 12px;
+}
+
+.headline-address h3 {
+  font-size: 30px;
+  font-weight: 900;
+  color: #272a31;
+  margin-bottom: 20px;
+}
+
+.headline-address p {
+  font-size: 16px;
+}
+
+.about-headline-address {
+  text-align: center;
+  font-size: 12px;
+  line-height: 22px;
+}
+
+.main {
+  background: #272a31;
+}
+
+footer .container {
+  display: flex;
+  flex-direction: row;
+  gap: 100px;
+  padding: 40px;
+}
+
+.main .container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.partners hr {
+  align-self: center;
+  height: 2px;
+  background: #ec5242;
+  border: none;
+  width: 5%;
+}
+
+.main .container hr {
+  align-self: center;
+  height: 2px;
+  background: #ec5242;
+  border: none;
+  width: 5%;
+  margin-bottom: 30px;
+}
+
+.speakers .container .one {
+  align-self: center;
+  height: 2px;
+  background: #ec5242;
+  border: none;
+  width: 5%;
+  margin-bottom: 30px;
+}
+
+.partners h2 {
+  color: white;
+  font-size: 20px;
+}
+
+.main .container h2 {
+  color: white;
+  font-size: 20px;
+  margin: 30px 0 10px 0;
+}
+
+.main .cards {
+  display: flex;
+  flex-direction: column;
+}
+
+.main .card {
+  display: flex;
+  flex-direction: row;
+  gap: 18px;
+  align-items: center;
+  padding: 20px;
+  color: white;
+  background: rgba(255, 255, 255, 0.1);
+  margin-bottom: 10px;
+}
+
+.main .card h3 {
+  color: #ec5242;
+}
+
+.main .card p {
+  font-size: 12px;
+}
```

Comment:
- [ ] As mentioned by the previous reviewer the cards in the mobile version of `main section` are not aligned properly. Please reference the comparison screen shots between the Cindy shin design and your design below and fix the alignment in your mobile version

| Cindy Shin design    | Your design |
| ------------- | ------------- |
| <img width="338" alt="Screenshot 2023-04-20 at 12 17 47 PM" src="https://user-images.githubusercontent.com/44798044/233291215-8cb24100-772c-41fc-aff3-2d6ea10bb59e.png"> | <img width="338" alt="Screenshot 2023-04-20 at 11 36 24 AM" src="https://user-images.githubusercontent.com/44798044/233291269-1e489538-da9c-4a22-b1a2-223079d523f6.png"> |